frontend/src/app/core/browser/device.service.ts

```diff
@@ -2,14 +2,19 @@ import { Injectable } from '@angular/core';
 
 @Injectable({ providedIn: 'root' })
 export class DeviceService {
-  public mobileWidthTreshold = 544;
-  public tabletWidthTreshold = 768;
+  public mobileWidthThreshold = 544;
+  public tabletWidthThreshold = 768;
+  public smallDesktopWidthThreshold = 1012;
 
   public get isMobile():boolean {
-    return (window.innerWidth < this.mobileWidthTreshold);
+    return (window.innerWidth < this.mobileWidthThreshold);
   }
 
   public get isTablet():boolean {
```

Comment:
isTablet and tabletWidthThreshold can be removed, since they are not used anymore.